Solutions/Infoblox Cloud Data Connector/Analytic Rules/Infoblox-HighNumberOfNXDOMAINDNSResponsesDetected.yaml

```diff
@@ -0,0 +1,43 @@
+id: 818eddaa-3806-43a2-8930-3defc5a06803
+name: Infoblox - High Number of NXDOMAIN DNS Responses Detected
+description: |
+  'This creates an incident in the event a high number of DNS responses for non-existent domains is generated by a single host.'
```

Comment:
Please mention time interval binning used and threshold in description instead of high number for clarity. 
e.g. Default : More than 200 in 15 mins which is customizable.

Reply:
@ashwin-patil Gotcha. I decided to remove the bin altogether and rely on the rule period. **Question 1:** I also added the GitHub link to the kusto parser (InfobloxCDC) required for the analytic rules in their descriptions -  https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Infoblox%20Cloud%20Data%20Connector/Parsers/InfobloxCDC.txt. This doesn't exist yet but I assume will once this PR gets merged. Is this the correct approach? I dug around other solutions on GitHub and noticed some parser links are the full GitHub url like above and some have the format "https://aka.ms/sentinel-<parsername>". I did the same thing with the data connector and workbook. **Question 2:** Will the InfobloxCDC parser be part of the solution so users don't have to copy and paste? I downloaded and noticed Corelight's solution came with the parser - didn't have to copy and paste. I'd like to do the same.